docs/api.rst

```diff
@@ -844,6 +844,20 @@ The path is encoded if needed.
 
    .. versionadded:: 1.9
 
+.. method:: URL.__truediv__(url)
+
+   Shortcut for :meth:`URL.joinpath` with a single element and ``encoded=False``.
+
+   .. doctest::
```

Comment:
Nice! I think this is the first doctest in the project. Though, we'd probably need to put a doctest sphinx invocation into CI for it to actually be checked…